src/Docker/Compose/DeveloperCompose.php

```diff
@@ -14,16 +14,23 @@
  */
 class DeveloperCompose extends ProductionCompose
 {
+    const SYNC_ENGINE_DOCKER_SYNC = 'docker-sync';
+    const SYNC_ENGINE_MUTAGEN = 'mutagen';
+    const SYNC_ENGINE = 'sync-engine';
```

Comment:
Is this one default sync?

Reply:
`SYNC_ENGINE = 'sync-engine'` - it is only a key
The default value is `docker-sync`.